src/modules/addComment.js

```diff
@@ -0,0 +1,44 @@
+import countComment from './countComment.js';
+
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Y9ST6EsEV41gcCvlbOfZ/comments';
+const comments = document.querySelector('.content');
+
+const addComment = () => {
```

Comment:
- [ ] [REQUIRED] You should fulfill the requirement that the [previous code reviewer](https://github.com/NickEmma/capstone-project-kanban/pull/29/files#r932475382) said, but you still haven't fulfilled it. Please do it!

![image](https://user-images.githubusercontent.com/63915024/181605051-11c35b02-9463-4bf2-aa17-c9c3e6718a56.png)

Reply:
The previous code reviewer said that the pop-up to stay on top of the section which I did